test/utils/helperFn.ts

```diff
@@ -55,13 +55,15 @@ export async function sleep(duration: number) {
 }
 
 export function getColumnValues(colIndex: number = 0) {
-  return Array.from(document.querySelectorAll(`[role="cell"][aria-colindex="${colIndex}"]`)).map(
+  return Array.from(document.querySelectorAll(`[role="cell"][data-colindex="${colIndex}"]`)).map(
     (node) => node!.textContent,
   );
 }
 
 export function getColumnHeaderCell(colIndex: number): HTMLElement {
-  const columnHeader = document.querySelector(`[role="columnheader"][aria-colindex="${colIndex}"]`);
+  const columnHeader = document.querySelector(
+    `[role="columnheader"][aria-colindex="${colIndex + 1}"]`,
```

Comment:
Here too?

Reply:
This is `column header`, not `cell`. So no `data-colIndex` for this, right? Check this comment - https://github.com/mui-org/material-ui-x/pull/1669#discussion_r632935645

Reply:
I would prefer to use the `data-colindex` everywhere as it would be more consistent with the code and the cell element

Reply:
First thing - are we considering `aria` or `data` after @oliviertassinari 's review yesterday. What conclusion we have made which would be better for us _clearly_? 
As for using `data-colindex: colIndex` for `ColumnHeader` as well, then the row cell element will be having starting base co-ordinates as `{rowIndex}-{colIndex}` -- `0-0`. And column header will be `data-colindex` -- `0` i.e now the header base is zero-based instead of one-based like previously. Are we good?